explore/circadian-efficacy/README.md

```diff
@@ -10,3 +10,25 @@ Evaluating unsupervised DWPC p-values and path contributions to predict whether
 + [`downloads/aat8806_Data_file_S1.xlsx`](downloads/aat8806_Data_file_S1.xlsx) contains data from CircaDB, which provides tissue-specific rhythmical scores for human genes. 
   The data was downloaded from Data file S1 of [A database of tissue-specific rhythmically expressed human genes has potential applications in circadian medicine](https://doi.org/10.1126/scitranslmed.aat8806).
   The algorithm 'CYCLOPS', which was developed to score genes based on circadian rhythms, can be accessed from [CYCLOPS reveals human transcriptional rhythms in health and disease](https://doi.org/10.1073/pnas.1619320114).
+
++ [`downloads/drug_links.csv`](downloads/drug_links.csv) contains mapping between compound names and DrugBank IDs, as well as other external links.
+  The data was downloaded from [External Drug Links in Drugbank](https://www.drugbank.ca/releases/5-1-3/downloads/all-drug-links).
```

Comment:
What do we use `drug_links.csv` for? I am worried this is not the right place to process or store drugbank cross-references.